data/blog/cadence-pause.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Upcoming Electron Releases
+author: codebytere
+date: '2019-05-13'
```

Comment:
wrong date ⏰

Reply:
we'd run into that issue where the blog wasn't showing up on the website, when we published E5's blog (if you remember that @codebytere ) 😂